ping.c

```diff
@@ -1618,3 +1616,28 @@ void ping4_install_filter(socket_st *sock)
 	if (setsockopt(sock->fd, SOL_SOCKET, SO_ATTACH_FILTER, &filter, sizeof(filter)))
 		error(0, errno, _("WARNING: failed to install socket filter"));
 }
+
+void print_header(int family, char *hostname, char *source, char *whereto, uint32_t flowlabel)
+{
+	printf("PING %s(%s) ", hostname, whereto);
```

Comment:
Like it or not people parse command outputs. Can you explain very carefully why ABI breakage is needed so that the future complaints can be directed to that text, and they convince people whose scripts, monitoring, and who knows what is broken that this was indeed great idea.

Reply:
Good point about output being parsed. I wonder if people noticed this behavior. But agree, whether it was deliberate to keep space for parsing or not, maybe some people depend on this formatting.

I might prepare another PR for other changes in this PR (Don't translate PING and reduce duplicity with `print_header()`, which will keep current formatting).

Ad parsing - weird idea: have also json output format (`ip` from [iproute2](https://git.kernel.org/pub/scm/network/iproute2/iproute2.git)) supports it: `ip --json addr`)